libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,275 @@
+  background-color: #FAFAFA;
+}
+
+.nav-marquee .content-table .toc-container {
+  flex: 1 1 auto; 
+  padding: 24px;
+}
+
+.nav-marquee .content-table .toc-title {
+  font-size: var(--type-heading-xs-size);
+  margin: 0;
+  font-weight: bold;
+}
+
+.nav-marquee .toc-list {
+  display: flex;
+  flex-direction: column; 
+  gap: 24px;
+  margin-top: 24px;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.nav-marquee .toc-item {
+  min-width: 200px;
+  display: flex;
+  position: relative; 
+  justify-content: space-between;
+  background-color: #FFF;
+  border: 1px solid #4B4B4B;
+  border-radius: 4px;
+  padding: 24px;
+  cursor: pointer;
+  line-height: 1.5;
+}
+
+.nav-marquee .toc-item:hover {
+  box-shadow: 2px 4px 5px #0000001A;
+  border: 1px solid #1473E6;
+}
+
+.nav-marquee .toc-item:focus-within {
+  border: 1px solid #1473E6;
+  background-color: #F5F9FD;
+}
+
+.nav-marquee .marquee p {
+  margin: 0 0 var(--spacing-s);
+}
+
+.nav-marquee .marquee .foreground .icon-area img{
+  height: 20px;
+  float: left;
+  padding-right: 16px;
+ }
+
+.nav-marquee .toc-link-text {
+  flex: 1 1 auto;
+}
+
+.nav-marquee .toc-item a {
+  color: var(--text-color);
+  outline: none;
+}
+
+.nav-marquee .toc-item a:hover {
+  text-decoration: none;
+}
+
+.nav-marquee .toc-link-text p:last-child {
+  margin-bottom: 0;
+}
+
+.nav-marquee .section-title {
+  font-weight: bold;
+  font-size: var(--type-body-s-size);
+  line-height: 20px;
+}
+
+.nav-marquee .section-description {
+  font-size: 14px;
+}
+
+.nav-marquee .toc-arrow {
+  align-self: center;
+  padding-left: 8px;
+}
+
+.nav-marquee .marquee {
+  min-height: unset;
+}
+
+.nav-marquee .marquee .foreground {
+  width: var(--grid-container-width);
+  margin: 0 auto;
+  padding-top: 56px;
+  padding-bottom: 0px;
+}
+
+.nav-marquee .marquee .foreground .media {
+  width: 100%;
+}
+
+@media screen and (min-width: 600px) {
+  .nav-marquee .content-table .toc-list {
+     flex-flow: row wrap;
+  }
+
+  .nav-marquee .toc-item {
+    flex: 1 1 45%; 
+    padding: 16px;
+  }
+
+  .nav-marquee .marquee .foreground .text h2 {
+    font-size: var(--type-heading-xxl-size);
+    clear: both;
+  }
+
+  .nav-marquee .marquee .foreground .text h3 {
+    font-size: var(--type-heading-m-size);
+    text-transform: none;
+  }
+
+  .nav-marquee .marquee .foreground .media {
+    margin: 0;
+  }
+}
+@media screen and (max-width: 600px) {
+  .nav-marquee .toc-list {
+    gap: 16px;
+  }
+
+  .nav-marquee .toc-item {
+    padding: 16px;
+  }
+
+  .nav-marquee .section-description {
+    font-size: 12px;
+    margin-top: 16px;
+  }
+
+  .nav-marquee .toc-footer{
+    font-size: 12px;
+  }
+
+  .nav-marquee .marquee .foreground {
+    padding-top: 48px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .section .nav-marquee {
+      display: flex;
+      flex-direction: row;
+  }
+
+  .nav-marquee .content-table {
+    background-color: #FFF;
+    border: none;
+  }
+
+  .nav-marquee .section-title {
+    line-height: 22.5px;
```

Comment:
Mixed usage for line-height:
1. unitless value - https://github.com/adobecom/milo/pull/786/files#diff-3052973bc600cc270df3e74fc90fefef74cfa0de6424aea73b1b415ea91e58a7R225
2. px value - https://github.com/adobecom/milo/pull/786/files#diff-3052973bc600cc270df3e74fc90fefef74cfa0de6424aea73b1b415ea91e58a7R175

Reply:
We typically prefer the unit less value. It tends to be less fussy and MDN recommends it.